2-mandatory/4-space-colonies.js

```diff
@@ -12,10 +12,17 @@
         Input: Is an array
         Output: Is an array
         Logic: Only strings that start with A, and finish with family
-  
+  npx jest --testRegex='4-space-colonies.js'
 */
 
-function getSettlers() {}
+function getSettlers(array) {
+
+ return array.filter(str => 
+    str.split(' ').pop()==="family"
```

Comment:
Nice! Do you know of another way to find out if a string ends with a certain word?